### PR TITLE
new travis tests

### PR DESCRIPTION
**Pull request by bgruening**
ping @joachimwolff 
This should work and the failures looks warrantable.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,45 +1,70 @@
...
+#  #- osx
+
+addons:
+  apt:
+    packages:
+    - samtools
+
+install:
+  # install planemo for Galaxy tool testing
+  - pip install flake8 planemo
```
Comment (joachimwolff) [inline .travis.yml]: Why do you use pip to install flake8?

File: .travis.yml [hunk 2]
```diff
@@ -1,45 +1,70 @@
...
-# command to run tests
+  # Replace dep1 dep2 ... with your dependencies
+  - conda build ./scripts  -c conda-forge -c bioconda
+  - conda create -y -n $HIC_CONDA_ENV python=$TRAVIS_PYTHON_VERSION
+  - conda create -y -n __samtools@1.3.1 samtools=1.3.1 -c bioconda
+  - source activate $HIC_CONDA_ENV
+  - conda install --yes --file requirements.txt -c conda-forge -c bioconda
+  - conda install --yes --use-local hicexplorer -c conda-forge -c bioconda
+  # isntall HiCExplorer into the conda environment
+  - pip install nose
```
Comment (joachimwolff) [inline .travis.yml]: Again, why pip and not conda?

File: .travis.yml (hunk 1 above)
Comment (bgruening) [inline .travis.yml]: It doesn't matter, you can also use conda. Important is just that you install planemo with pip.

File: .travis.yml (hunk 2 above)
Comment (bgruening) [inline .travis.yml]: Some here, you can probably also use conda.

**Comment by joachimwolff** [comment]
I merged these changes in #116 and created a PR for it. Maybe we can close this PR because the changes will come to develop and soon to master anyway?

**Comment by bgruening** [comment]
Sure.

**Pull request closed without merging**